src/open_inwoner/accounts/views/cases.py

```diff
@@ -423,30 +429,28 @@ def handle_document_upload(self, request, form):
 
         file = cleaned_data["file"]
         title = cleaned_data["title"]
-        user_choice = cleaned_data["type"].id
+        document_type = cleaned_data["type"]
```

Comment:
The ModelChoiceField returns an instance of the object we want, so no need to take its ID and then later do a `.get()` to retrieve a copy of that same object.